_inc/client/traffic/verification-services.jsx

```diff
@@ -16,11 +16,30 @@ import {
 import { ModuleSettingsForm as moduleSettingsForm } from 'components/module-settings/module-settings-form';
 import SettingsCard from 'components/settings-card';
 import SettingsGroup from 'components/settings-group';
+import JetpackBanner from 'components/jetpack-banner';
 
 export const VerificationServices = moduleSettingsForm(
 	React.createClass( {
+		activateVerificationTools: function() {
+			return this.props.updateOptions( { [ 'verification-tools' ]: true } );
+		},
+
 		render() {
 			const verification = this.props.getModule( 'verification-tools' );
+
+			// Show one-way activation banner if not active
+			if ( ! this.props.getOptionValue( 'verification-tools' ) ) {
+				return (
+					<JetpackBanner
+						callToAction={ __( 'Activate' ) }
+						title={ verification.name }
+						icon="cog"
+						description={ verification.long_description }
+						onClick={ this.activateVerificationTools }
+					/>
```

Comment:
It's the same with the search banners.  
![screen shot 2017-08-08 at 3 46 35 pm](https://user-images.githubusercontent.com/7129409/29091388-cea70b50-7c50-11e7-88cd-1a67978bc290.png)

Likely something going on with the `<Banner />` dops-component